modules/qt/nixos.nix

```
@@ -28,6 +28,9 @@ in
   };
 
   config = lib.mkIf (config.stylix.enable && config.stylix.targets.qt.enable) {
+    warnings = lib.mkIf config.services.xserver.desktopManager.plasma6.enable [
+      "Styling of Plasma6 is not yet supported upstream. See https://github.com/nix-community/home-manager/issues/5098"
```

The warning should follow the standardized format from https://github.com/danth/stylix/commit/5e8be7521e8f7cc927c92f24f8753e64ec050cdf. (Which admittedly is not yet documented...)

```suggestion
      "stylix: kde: styling of Plasma 6 is not yet supported by Home Manager: see https://github.com/nix-community/home-manager/issues/5098"
```

I also suggest replacing "upstream" with "Home Manager" to make it clearer what we're referring to.

---

In this case upstream is both home-manager and nixpkgs.
The reason I used `mkDefault` is because I wanted to make it easier for users to override the value because most will wish to do so. Qt styling so far has been only implemented for the minority that does not use a popular Desktop environment.
I will change the commit message.